docs/src/intro-csharp.md

```diff
@@ -73,12 +72,56 @@ By default, Playwright runs the browsers in headless mode. To see the browser UI
 await playwright.Firefox.LaunchAsync(new BrowserTypeLaunchOptions { Headless = false, SlowMo = 50 });
 ```
 
+## First test
+
+You can choose to use NUnit test fixtures that come bundled with Playwright. These fixtures support running tests on multiple browsers out of the box. They also allow running tests in parallel. Learn more about [Playwright with NUnit](./test-runners.md).
```

Comment:
"multiple browser" is talking about running in parallel, right?  Let's make it a bit more clear.